src/phaser-game/scenes/EndGame.ts

```diff
@@ -0,0 +1,28 @@
+import Phaser from 'phaser';
+
+class EndGame extends Phaser.Scene {
```

Comment:
uuu you're using class to build this interesting

Reply:
Yes, I know OOP is controversial, but the Phaser documentation is written with classes so wish me luck lol